test.js

```diff
@@ -190,30 +190,30 @@ describe('ns', function() {
 
   describe('metric', function() {
     it('should default to null if not a metric namespace', function() {
-      assert.equal(ns('canadian-things.foods').metric, null);
+      assert.equal(ns('canadian-things.foods').metric, undefined);
```

Comment:
should we just remove this test, as we are now no longer concerned about "metrics" ? and all other tests related to metrics?